react/Minicart.tsx

```diff
@@ -24,9 +37,10 @@ interface MinicartProps {
   drawerSlideDirection: SlideDirectionType
   quantityDisplay: QuantityDisplayType
   itemCountMode: MinicartTotalItemsType
-  backdropMode: MaybeResponsiveValue<BackdropMode>
+  backdropMode: ResponsiveValuesTypes.ResponsiveValue<BackdropMode>
   customPixelEventId: string
-  customPixelEventName: PixelData['event']
+  customPixelEventName: PixelEventTypes.EventName
+  classes: CssHandlesTypes.CustomClasses<typeof CSS_HANDLES>
```

Comment:
```suggestion
  classes?: CssHandlesTypes.CustomClasses<typeof CSS_HANDLES>
```

Reply:
This is not necessary, since I'm applying a `Partial` when using this type:

```tsx
const Minicart: FC<Partial<MinicartProps>> = ({ ...
```

Reply:
That's kind of unconventional. I would rather change it to be explicit on which props are optional.

Reply:
No problem! But every prop is optional 😅 
Using a Partial seemed like a cleaner way to say "everything is optional"